README.md

```diff
@@ -1 +1,134 @@
-# persistenceJS
+<p align="center" size="20">
+    <span style="color:black;font-weight:700;font-size:30px">
+    Persistence
+    </span> 
+    <img src="https://www.computerhope.com/jargon/j/javascript.png" alt="isolated" width="40"/>
+</p>
+<p align="center">
+    <a href="https://opensource.org/licenses/Apache-2.0"><img alt="License" src="https://img.shields.io/badge/License-Apache_2.0-blue.svg"></a>
+    <a href="https://www.npmjs.com/package/persistenceonejs"><img alt="npm (scoped)" src="https://img.shields.io/npm/v/persistenceonejs.svg"> </a>
+</p>
+
+<p align="center">
+    <a href="https://docs.pstake.finance/">Core</a>
+    ·
+    <a href="https://cosmos.github.io/cosmjs/latest/stargate/index.html">Docs</a>
+</p>
+
+---
+
+## Setup
+
+```sh
+npm install persistenceonejs
```

Comment:
should we rename the repo name itself to `persistenceonejs` from `persisitenceJS`